<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_Friends_Frame</class>
 <widget class="QWidget" name="Add_Friends_Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>185</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/resorces/resorces/login.jpg);</string>
  </property>
  <widget class="QTextEdit" name="display_result">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>185</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>54</width>
     <height>11</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索结果：</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>185</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>搜索账号</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="friend_id"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>180</y>
     <width>158</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button_sercher">
      <property name="text">
       <string>搜索</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_add">
      <property name="text">
       <string>添加</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
